content/applications/marketing/survey.rst

```diff
@@ -1,8 +1,266 @@
-======
-Survey
-======
+=======
+Surveys
+=======
 
-.. toctree::
-   :titlesonly:
+**Odoo Surveys** helps users create surveys, appraisals, quizzes, and questionnaires to gather
+valuable information from their customers or employees.
 
-   survey/overview
\ No newline at end of file
+.. seealso::
+   - `Odoo Surveys: product page <https://www.odoo.com/app/surveys>`_
+   - `Odoo Tutorials: Surveys <https://www.odoo.com/slides/surveys-62>`_
+
+Question configuration
+======================
+
+Add a question
+--------------
+
+When you click *Add a question* in the *Questions* tab on the survey template page, a pop-up
+appears, in which you create your question.
+
+After writing your question in the *Question* field, select a *Question Type.* A preview of the
+question type is displayed on the right side of the pop-up window.
+
+.. image:: survey/preview-question-type.png
+   :align: center
+   :alt: Selecting a question type in Odoo Surveys
+
+Question types and features
+---------------------------
+
+The following is a list of the available *Question Types* and their specific features:
+
+* **Multiple Lines Text Box** - open-text field suitable for long answers where any type of text
+  is allowed
+
+* **Single Line Text Box** - open-text field suitable for short answers where any type of text is
+  allowed
+
+* **Numerical Value** - open-text field that only accepts a number as an answer
+
+* **Date** - date selection on a calendar
+
+* **Datetime** - date and time selection on a calendar
+
+* **Multiple choice: only one answer** - closed question where only one answer can be selected
+
+* **Multiple choice: multiple answers allowed** - closed question where multiple answers can be
+  selected
+
+* **Matrix** - table of multiple choice questions that share the same possible answers
+
+Answers and Options tab
+-----------------------
+
+Features change in the *Answers* and *Options* tab. It all depends on the *Question Type*.
+
+.. note::
+   Some options are common to many *Question Types*, such as *Validate entry*, *Mandatory
+   Answer*, *Question Time Limit* (for *Live Sessions*), *Conditional Display*, and *Score/Scored*.
```

Comment:
```suggestion
Features change in the *Answers* and *Options* tab. It all depends on the *Question Type*. Below are presented options that are common to several *Question Types*, such as *Validate entry*, *Mandatory Answer*, *Question Time Limit* (for *Live Sessions*), and *Conditional Display*.
```
It didn't feel quite right to have that note, so I moved it to the main paragraph.
I removed the score/scored as it isn't described here.